flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/util/TableTestBase.scala

```diff
@@ -147,13 +161,15 @@ abstract class TableTestUtil(test: TableTestBase) {
     * @param types field types
     * @param fields field names
     * @param statistic statistic of current table
+    * @param uniqueKeys unique keys of current table
     * @return returns the registered [[Table]].
     */
   def addTableSource(
       name: String,
       types: Array[TypeInformation[_]],
       fields: Array[String],
-      statistic: FlinkStatistic = FlinkStatistic.UNKNOWN): Table
+      statistic: TableStats,
```

Comment:
we may add more info for `FlinkStatistic`, use `FlinkStatistic` instead of each fields to make sure this method and related test cases are stable.

Reply:
If the more information we need add to `FlinkStatistic`, I think it should also be included in `TableStats`. Regarding to the `relModifiedMonotonicity`, it is only be used internally in intermediate table source (`IntermediateRelTable`) which keeps `FlinkStatistic` as the constructor parameter.

Reply:
what's the problem using `FlinkStatistic` ?

Reply:
We removed the `FlinkStatistic` from the constructor of `TableSourceTable` to make the statistic deriving simple. That's why we can't build a `TableSourceTable` from `FlinkStatistic`.